install-devel.sh

```diff
@@ -183,12 +183,12 @@ print_env_vars() {
 # installation folder as much as possible
 conda_cleanup() {
     conda clean --index-cache
-    conda clean --lock
     conda clean --tarballs -y
-    conda clean --packages -y
+    conda update --all
```

Comment:
Is this really required here? 

Could you please let me know what are you trying to achieve?